python/cuml/dask/ensemble/base.py

```diff
@@ -92,17 +92,17 @@ def _concat_treelite_models(self):
         to create a single model. The concatenated model is then converted to
         bytes format.
         """
-        model_protobuf_futures = list()
+        model_serialized_futures = list()
         for w in self.workers:
-            model_protobuf_futures.append(
-                dask.delayed(_get_protobuf_bytes)
+            model_serialized_futures.append(
+                dask.delayed(_get_model_serialized)
                 (self.rfs[w]))
-        mod_bytes = self.client.compute(model_protobuf_futures, sync=True)
+        mod_bytes = self.client.compute(model_serialized_futures, sync=True)
         last_worker = w
         all_tl_mod_handles = []
         model = self.rfs[last_worker].result()
-        all_tl_mod_handles = [model._tl_model_handles(pbuf_bytes)
-                              for pbuf_bytes in mod_bytes]
+        all_tl_mod_handles = [model._tl_handle_from_bytes(x)
+                              for x in mod_bytes]

```

Comment:
a variable name like 'indiv_worker_model_bytes` or something which tells the person reading the code what mod_bytes contain

Reply:
Done